hack/update-bindata.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+KUBE_ROOT="../../../"
```

Comment:
This is done to keep it consistent with the paths in the ReadOrDie utility.  It definetly will work, and will die if its invoked incorrectly.    Minor refactor later if we want to use $KUBE_ROOT for everything, but as first commit i think this is more straightforward for anyone looking at the code, since it matches to the relative paths in the generators.